testData/solutions/win32.json5

```diff
@@ -1548,6 +1548,183 @@
         ]
     },
 
+    "com.microsoft.windows.fontSize": {
+        "name": "Windows Font Size",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure.NonClientMetrics": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "options": {
+                    "getAction": "SPI_GETNONCLIENTMETRICS",
+                    "setAction": "SPI_SETNONCLIENTMETRICS",
+                    "uiParam": "struct_size",
+                    "pvParam": {
+                        "type": "struct",
+                        "name": "NONCLIENTMETRICS"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE", // It won't update if these flags aren't set.
+                    "verifySettings": false // The next GET will be different to the last SET.
```

Comment:
Updated comment..
`// The next GET will be different to the last SET, because the value of lfHeight sets the character
// height (negated), but receives it as the cell height.`

LOGFONT: https://msdn.microsoft.com/library/dd145037

Not only is the value different, it also uses the sign to specify if you're talking about the character height (-) or cell height (+).

More info: https://support.microsoft.com/en-us/help/32667/info-font-metrics-and-the-use-of-negative-lfheight

So why not just specify the cell height? The preference is already the character height, and calculating the cell height requires knowing the DPI, which is a run-time value.

I think the LOGFONT structure is from when raster fonts where common.